app/global.R

```diff
@@ -87,25 +87,21 @@ conditions_list = c("Chronic renal disease" = "renal_disease",
                     "Other chronic condition" = "other")
 
 # activities list placebolder 
-l_activities_list = c("grocery shopping", 
-                      "library or museum",
-                      "outdoor socially distanced exercise (e.g. tennis, golf, bike riding)", 
-                      "outdoor visit with friends")
-m_activities_list = c("children’s playdate and playgrounds",
-                      "indoor public spaces (e.g. salon, casino, bowling, shopping mall)",
-                      "public beach or pool",
-                      "public transportation/airplane",
-                      "overnight stay at a hotel", 
-                      "outdoor dinning",
-                      "see doctor or dentist", 
-                      "school, camp, or daycare",
-                      "working outside of home")
-h_activities_list = c("indoor restaurant or bar", 
-                      "nightclub/music concert/movie theater/gym",
-                      "sports stadium",
-                      "religious services (including weddings and funerals)", 
-                      "team sports (basket ball, football)", 
-                      "visiting nursing home or hospital")
+l_activities_list = c("Grocery shopping", 
+                      "Visiting library or museum",
+                      "Outdoor socially distanced activities")
+m_activities_list = c("Indoor public spaces (e.g. salon, casino, bowling, shopping mall)",
+                      "Public transportation/airplane",
+                      "Overnight stay at a hotel", 
+                      "Outdoor dining",
+                      "Seeing doctor or dentist",
```

Comment:
Do you think the risk from this activity comes from direct contact with the doctor? If so should we remove?

Reply:
I think the risk comes from a combination of direct contact with the doctor and other hospital staff, the waiting room, being in a small space, and commonly touched surfaces.